showyourwork/workflow/scripts/pdf.py

```diff
@@ -78,3 +78,9 @@
         str(paths.user().compile / (snakemake.config["ms_name"] + ".pdf")),
         str(Path(snakemake.config["ms_pdf"])),
     )
+
+    # Copy the synctex file (if present) to the user dir
+    for file in [".synctex", ".synctex.gz"]:
```

Comment:
I would check the .gz file first, and, if it succeeds, break. The reason is that there may be both files - the .gz file should be preferred since it’s smaller.

Reply:
Good idea!